azhlfTool/common/AdminProfileManager.ts

```diff
@@ -0,0 +1,38 @@
+import { join as joinPath } from "path";
+import { readFile, writeFile, pathExists, mkdirp } from "fs-extra";
+import { AdminProfile } from "./Interfaces";
+import { Constants } from "./Constants";
+import { ObjectToString } from "./LogHelper";
+
+export class AdminProfileManager {
+    private readonly profilesDirectory: string;
+
+    public constructor(){
+        this.profilesDirectory = joinPath(Constants.StoresPath, "adminprofiles");
+    }
+
+    public async getAdminProfile(organization: string): Promise<AdminProfile> {
+        const adminProfilePath = joinPath(this.profilesDirectory, `${organization}_AdminCredential.json`);
```

Comment:
The filename formatting between getadminprofile and writeAdminprofile are different, is that by design.

Reply:
I think it is 2 different variables. In case of getAdminProfile (which is implicitly not used as of now), the file will be read based user input of `organization` (member name).

In case of writeAdminProfile (which is used by the import admin credentials command), the function takes one argument, admin profile, and uses its property: `msp_id` (which is implicitly same as organization name), for storing or writing the file.